mypyc/test-data/irbuild-i64.test

```diff
@@ -666,13 +665,11 @@ L0:
     return r0
 def g():
     r0 :: tuple[int, int]
-    r1 :: object
-    r2 :: tuple[int64, int64]
+    r1 :: tuple[int64, int64]
 L0:
     r0 = (2, 4)
```

Comment:
It's a little unfortunate that this remains in the IR, but it's nothing DCE can't fix if we ever implement it.

```c
tuple_T288 CPyDef_g(void) {
    tuple_T2II cpy_r_r0;
    tuple_T288 cpy_r_r1;
CPyL0: ;
    cpy_r_r0.f0 = 2;
    cpy_r_r0.f1 = 4;
    CPyTagged_INCREF(cpy_r_r0.f0);
    CPyTagged_INCREF(cpy_r_r0.f1);
    CPyTagged_DECREF(cpy_r_r0.f0);
    CPyTagged_DECREF(cpy_r_r0.f1);
    cpy_r_r1.f0 = 1;
    cpy_r_r1.f1 = 2;
    return cpy_r_r1;
}
```

Reply:
The C compiler may optimize some of these away. Also, optimizing these shouldn't be too difficult in mypyc. Added https://github.com/mypyc/mypyc/issues/983 to track this.